mlrun/feature_store/steps.py

```diff
@@ -634,3 +637,9 @@ def _do_pandas(self, event):
 
     def _do_spark(self, event):
         return event.drop(*self.features)
+
+    def validate(self, entities: List[str] = None):
+        if set(self.features).intersection(entities):
+            raise mlrun.errors.MLRunInvalidArgumentError(
+                "DropFeatures can only drop features, not entities"
```

Comment:
It's better usability to show the offending features here - since you already calculate the intersection, just show it here.

Reply:
fixed! tnx